### PR TITLE
Update GitHub action to not comment about visual regression tests if build fails

### PR DESCRIPTION
**Pull request by JamesBurnside**
# What
Quick fix to ensure github action only comments about visual regression tests when the tests fail and not the build fails

# Why
When the packages build fails you would get a PR comment saying that you should update the UI tests. This comment should only happen if the _tests_ fail and not if the build step fails

# How Tested
Many CI runs, verified:
* All passes - no comment
* Build failes - no comment
* Build passes but tests fail - adds the comment

### CODE REVIEW

**Review by PorterNan — APPROVED** [review]
Thanks for improving this one!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -160,19 +160,20 @@ jobs:
       - name: Build
         run: rush build -t sample-static-html-composites
       - name: Visual Regression Tests
+        id: visualregressiontests
```
Comment (PorterNan) [inline .github/workflows/ci.yml:163]: Do we wanna add - between words

Comment (JamesBurnside) [reply .github/workflows/ci.yml:163]: I just wasn't confident that github actions wouldn't break when doing `steps.visual-regression-tests.outcome` 😅

Comment (JamesBurnside) [reply .github/workflows/ci.yml:163]: (completing without making this change as keeping this to a v. quick fix and waiting for the ci to run a couple times to test is just too much 😭)

**Pull request merged**